src/Data/FormURLEncoded.purs

```diff
@@ -29,9 +30,9 @@ instance showFormUrlEncoded :: Show FormURLEncoded where
   show (FormURLEncoded kvs) = "(FormURLEncoded " <> show kvs <> ")"
 
 -- | Encode `FormURLEncoded` as `application/x-www-form-urlencoded`.
-encode :: FormURLEncoded -> String
-encode = String.joinWith "&" <<< map encodePart <<< toArray
+encode :: FormURLEncoded -> Maybe String
+encode = map (String.joinWith "&") <<< sequence <<< map encodePart <<< toArray
   where
     encodePart = case _ of
-      Tuple k Nothing -> unsafeEncodeURIComponent k
-      Tuple k (Just v) -> unsafeEncodeURIComponent k <> "=" <> unsafeEncodeURIComponent v
+      Tuple k Nothing -> encodeURIComponent k
+      Tuple k (Just v) -> (\k v -> k <> "=" <> v) <$> encodeURIComponent k <*> encodeURIComponent v
```

Comment:
This produces compiler warnings due to name shadowing for `k` and `v`. What about this?

```suggestion
      Tuple k (Just v) -> (\key val -> key <> "=" <> val) <$> encodeURIComponent k <*> encodeURIComponent v
```

Reply:
Oops, great catch. I went directly into PSCi and never saw the warning!